shovel-config-ts/src/index.ts

```diff
@@ -0,0 +1,131 @@
+type Hex = `0x${string}`;
+
+export type PGColumnType =
+  | "bool"
+  | "bytea"
+  | "int"
+  | "numeric"
+  | "text"
+  | "timestamp";
```

Comment:
Is this type correct? The docs suggest otherwise https://github.com/indexsupply/code/blob/949f0fbabba96d2bacfb0beb12f395177da201ed/indexsupply.com/shovel/docs/index.md?plain=1#L306